docs/recipes/debugging-with-node-inspect.md

```diff
@@ -0,0 +1,24 @@
+# Debugging tests with node-inspect
+
+Because ava exectes its tests in parallel over multiple worker processes, it doesn't play nice with node's inspect feature by default.
```

Comment:
```suggestion
Because AVA executes its tests in parallel over multiple worker processes, it doesn't play nice with node's inspect feature by default.
```